pom.xml

```diff
@@ -68,16 +68,19 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
-            <version>1.2.17</version>
-            <scope>provided</scope>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+            <version>2.16.0</version>
```

Comment:
I believe this is a compile time from the slf4j binding so no need to explicitly add it. Also can we can go to 2.17.1? There are still vulns in 2.16 (and in 2.17.1 but aren't fixed yet)